src/ngraph/op/relu.cpp

```diff
@@ -24,38 +24,23 @@ op::Relu::Relu(shared_ptr<Node> arg)
     : UnaryElementwiseArithmetic("Relu", {arg})
 {
     constructor_validate_and_infer_types();
-    set_output_type(0, arg->get_element_type(), arg->get_shape());
```

Comment:
NB: I dropped what I think is a redundant `set_output_type` here (`Relu` is in `UnaryElementwiseArithmetic`).